client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -126,8 +127,8 @@ const Header: React.FunctionComponent = () => {
 	);
 
 	const handleCreateSite = React.useCallback(
-		( username: string, bearerToken?: string, planSlug?: string ) => {
-			createSite( username, freeDomainSuggestion, bearerToken, planSlug );
+		( username: string, bearerToken?: string, isPublicSite?: boolean ) => {
+			createSite( username, freeDomainSuggestion, bearerToken, isPublicSite );
```

Comment:
When checking all the places where we're dispatching `createSite`, I see there is one more in which we aren't providing the `isPublicSite` argument: https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/onboarding-block/edit.tsx#L80

It seems to called after a redirect to login based on `?new` query param.